plugins/scanners/scancode/src/main/kotlin/ScanCodeResultModel.kt

```diff
@@ -19,11 +19,14 @@
 
 package org.ossreviewtoolkit.plugins.scanners.scancode
 
+import java.io.File
+
 import kotlinx.serialization.Serializable
+import kotlinx.serialization.SerializationException
 import kotlinx.serialization.json.JsonArray
 import kotlinx.serialization.json.JsonElement
-import kotlinx.serialization.json.JsonTransformingSerializer
-import kotlinx.serialization.serializer
+import kotlinx.serialization.json.JsonPrimitive
+import kotlinx.serialization.json.jsonPrimitive
```

Comment:
commit-message: Please provide a rationale, why this is needed.

Reply:
The functionality is being used in the next commit, but I've added an explanation nonetheless.